tools/BUILD

```diff
@@ -3,4 +3,5 @@ package(default_visibility = ["//visibility:public"])
 exports_files([
     "gen_git_sha.sh",
     "git_sha_rewriter.py",
+    "socket_passing.py",
```

Comment:
This should now be declared with `envoy_py_test_binary` (hot off the press, since #823, which is still inflight but will probably be merged before this PR).